pages/index.js

```diff
@@ -1,17 +1,9 @@
 import React from 'react';
 import Layout from '../components/Layout';
 import EventList from '../components/EventList';
-import EVENTS from '../fixtures/events'
 
 export default class Index extends React.Component {
-  static async getInitialProps () {
```

Comment:
Pages which have the `getInitalProps` method are the only place we can make a server request for events. If we are serving a static html page on the initial request we need to make the dispatch here. We can one of two things, 1) render the event listing in "pages/index" so we remove the eventList component or 2) pass the events down to the component. I prefer the first option.